src/ansible_navigator/configuration_subsystem/navigator_post_processor.py

```diff
@@ -102,9 +102,10 @@ class NavigatorPostProcessor:
     """application post processor"""
 
     def __init__(self):
-        # Volume mounts accumulated from post processing various config entries.
-        # These get processed towards the end, in the (delayed)
-        # execution_environment_volume_mounts() post-processor.
+        """Initialize the post processor."""
+        #: Volume mounts accumulated from post processing various config entries.
+        #: These get processed towards the end, in the (delayed)
+        #: execution_environment_volume_mounts() post-processor.
```

Comment:
Looks like this should be
```suggestion
        #: ``execution_environment_volume_mounts()`` post-processor.
```

or a proper RST role `:meth:` reference.

Reply:
proper meth added cool.

Reply:
for anyone else, this is how it looks with a link to the function 
![image](https://user-images.githubusercontent.com/18386516/150585150-d1a20fd6-593f-48b2-8313-6774db15ffbf.png)